airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -1805,7 +1805,8 @@ public Stream<? extends Arguments> provideArguments(final ExtensionContext conte
           .filter(testCase -> testCase.get("enabled").asBoolean())
           .map(testCase -> Arguments.of(
               testCase.get("id").asText(),
-              testCase.get("namespace").asText(),
+              // Randomise namespace to avoid collisions between tests.
+              Strings.addRandomSuffix(testCase.get("namespace").asText(), "", 5),
```

Comment:
Will this leave any randomly names schemas or tables in the database at the end of the test?  I don't think so...

Reply:
Good question - I actually think we already do so today.. it doesn't look like the Redshift Acceptance Test cleans up after itself for [this particular case](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java#L103).

Reply:
added!

Reply:
Looks like [Snowflake](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestinationAcceptanceTest.java#L188) and Redshift are set up to erase entries in the TEST_SCHEMA.

Reply:
Ah looks like you added this: https://github.com/airbytehq/airbyte/pull/29312/files